src/panels/lovelace/cards/hui-shopping-list-card.ts

```diff
@@ -0,0 +1,279 @@
+import { html, LitElement } from "@polymer/lit-element";
+import { repeat } from "lit-html/directives/repeat";
+import "@polymer/paper-checkbox/paper-checkbox.js";
+import "@polymer/paper-icon-button/paper-icon-button.js";
+import "@polymer/paper-input/paper-input.js";
+import "@polymer/paper-item/paper-icon-item.js";
+import "@polymer/paper-item/paper-item-body.js";
+import "@polymer/paper-fab/paper-fab.js";

```

Comment:
absolutely no fab. It draws all the attention on the screen. And material design guidelines are saying not to use it in cards.

Reply:
Revert, Revert

Reply:
it didn't feel great while adding it :)

Reply:
import is still there.